### PR TITLE
Replace carrier_notes to carrier_note

### PR DESCRIPTION
**Pull request by nan-tic-dev**
#168436

### CODE REVIEW

**Review by bernatnan — APPROVED** [review]
LGTM

File: shipment.py [hunk 1]
```diff
@@ -281,7 +282,10 @@ def send_nacex(cls, api, shipments):
             data['tel_ent'] = unspaces(shipment.customer.mobile or
                 shipment.customer.phone or '')[:15]
             if shipment.carrier_note:
-                data['obs1'] = unaccent(shipment.carrier_note)[:38].rstrip()
+                blocks = split_into_blocks(unaccent(shipment.carrier_note).rstrip(), max_length=38)
```
Comment (bernatnan) [inline shipment.py]: PEP8 ¿? width = 79 or 80.

**Review by bernatnan — CHANGES_REQUESTED** [review]

File: shipment.py (hunk 1 above)
Comment (raimonesteve) [reply shipment.py]: See 4c3d32ab9da93b7c5c08b648f5a9881f89f5fc35

**Review by bernatnan — CHANGES_REQUESTED** [review]
> Only one tab for each parenthesis.

See 1519d78be6b0bb16004f3f9bdc870b5a205988e1

**Pull request merged**